### DISCUSSION TITLE
Some skin problem

### DISCUSSION THREAD

**Discussion by rXelelo** [ppy/osu#28906] [Q&A]
### Type

Cosmetic

### Bug description

When i change ui scale skin cracking

### Screenshots or videos

[Запис з 2024-07-18 11-11-52.webm](https://github.com/user-attachments/assets/e941ccc2-4393-4e70-bb2e-292f758ee77b)

### Version

2024.718.0

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16279901/compressed-logs.zip)

**Comment by peppy** [comment]
Your skin is probably doing hacky things which won't support UI scaling. Find another skin or use 100% scale.

**Reply by rXelelo** [reply]
:(

**Reply by rXelelo** [reply]
peppy can you in next update add editing skin textures  pls🥺

**Reply by Stoppedpuma** [reply]
Ctrl + Shift + S to open the skin editor, you can add elements with the correct names and they should work. Additionally you should be able to to this externally next update according to the update video.

**Reply by rXelelo** [reply]
i know about skin editor ok i will wait next video update
